### DISCUSSION TITLE
Does Zed have a linter for Jinja HTML files?

### DISCUSSION THREAD

**Discussion by patgd** [zed-industries/zed#7957] [Feature Requests]
If not, are there plans to implement one in the future? I'm curious about Zed's ability to interpret Django projects.

**Comment by SomeoneToIgnore** [comment]
Zed does not have any Jinja-related, or even Django-related support specifically.

While things could change, Zed team itself does not have any plans to write the code to support any specific language — instead, part of the team is focused on supporting plugins (step by step, starting from themes and language extensibility features) and offload the language support on the community.

Also, part of the team is reviewing any language-related PRs, so the community could use Zed with more languages while the plugin system matures.
There's https://github.com/zed-industries/extensions already and, at certain point, we plan to move away all existing language support there in the form of plugins.

Thus being said, any future PRs that support Django are welcome, but no direct plans from the Zed team to make it happen.
